source/tutorials/failover_syslog_server.rst

```diff
@@ -12,16 +14,19 @@ Let's assume you have a primary and two secondary central servers. Then, you
 can use the following config file excerpt to send data to them:
 rsyslog.conf:
 
-::
+.. code-block:: none
+
+   action(type="omfwd" target="primary-syslog.example.com" port="15314"
+                protocol="tcp")
+   & action(type="omfwd" target="secondary-1-syslog.example.com" port="15314"
```

Comment:
The & is no longer needed and may even cause harm (not 100 sure) - did you run the config once through rsyslog?